inference-engine/tests/functional/plugin/cpu/single_layer_tests/resample.cpp

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
I am still against merging test based on legacy operation. Lets reimplement it using Interpolate operation. We can wait until [related changes](https://github.com/openvinotoolkit/openvino/pull/2366) in Interpolate operation will be merged to master. However in this case I would prefer to remove ResampleV2 test at all.

Reply:
Done